README.md

```diff
@@ -230,6 +230,36 @@ sudo systemctl enable et.service
 sudo systemctl start et.service
 ```
 
+### Raspbian (Stretch):
+## (_tested on_: Raspberry Pi 3B+):
+
++ Install dependencies
+
+	sudo apt install python3-protobuf protobuf-compiler
+
++ Install from source
+```
+mkdir ~/tmp
+cd ~/tmp
+git clone --recurse-submodules https://github.com/MisterTea/EternalTerminal.git
```

Comment:
Thanks for creating this PR, I was struggling to find the right way to install ET.
From your PR I was able to learn that offering `--recurse-submodules` is a key argument because otherwise Raspbian does not have the compatible package ET depends on.

Pulling the correct source is the key to installing this successfully locally. Thank you!